scapy/contrib/EtherCat.uts

```diff
@@ -0,0 +1,256 @@
+from enum import test
```

Comment:
What is this?

Reply:
fixed in ae0317c (I was fooled by pycharm. again. sorry.)